inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -165,6 +167,8 @@ static void Transformation(CNNNetwork& clonedNetwork, const Config& conf) {
     manager.register_pass<ngraph::pass::ConvertNMS3ToNMS5>();
     manager.register_pass<ngraph::pass::ConvertNMS4ToNMS5>();
     manager.register_pass<ngraph::pass::ConvertNMSToNMSIEInternal>();
+    manager.register_pass<ngraph::pass::ConvertMulticlassNmsToMulticlassNmsIE>();
```

Comment:
I don't understand how does CPU plugin support such operations?

Reply:
Currently CPU plugin does not support dynamic shape yet and ConvertNMSToNMSIEInternal/ConvertMulticlassNmsToMulticlassNmsIE... will set the dynamic shape to static shape, such as the 'selected_outputs' output shape is [[0, 123], 6], the shape will become [123, 6] after running the pass.